examples/cra-kitchen-sink/.storybook/config.js

```diff
@@ -20,3 +20,5 @@ function loadStories() {
 }
 
 configure(loadStories, module);
+
+export { getStorybook };
```

Comment:
getStorybook returns a hash-table of stories.

If the config would export this, we can make storyshots framework agnostic 🎉